apps/www/_blog/2023-04-10-o11y-day.mdx

```diff
@@ -0,0 +1,206 @@
+---
+title: Supabase Logs are now self-hosted
```

Comment:
```suggestion
title: Supabase Logs are now self-hosted
launchweek: 7
```

Reply:
@Ziinc Can you please just add this `launchweek: 7` to the meta data?
We'll use it to show some additional links at the end of the post.

Reply:
done